admin/src/ConflictDetectionScannerSection.js

```diff
@@ -27,37 +27,40 @@ export default function ConflictDetectionScannerSection() {
   }, [showConflictDetectionReporter])
 
   return <div>
-    <h1>Conflict Detection Scanner</h1>
-    <p className={sharedStyles['explanation']}>
-      If you are having trouble loading icons on your site, you can use this
-      conflict scanner to detect possible Font Awesome version conflicts coming
-      from themes or other plugins that are loading other versions of Font Awesome.
+    <h2 className={ sharedStyles['section-title'] }>Conflict Detection Scanner</h2>
+    <div className={sharedStyles['explanation']}>
+      <p>If you are having trouble loading Font Awesome icons on your site, use this
+      conflict scanner to detect possible conflicts coming from themes or other plugins
+      that are loading other versions of Font Awesome.</p>
 
-      After you enable the scanner, a scanner box will appear in the bottom corner
-      of your window (only you can see it) while it runs for 10 minutes. You can
-      then browse your site -- especially pages that are having trouble -- to
-      catch the conflicts in the scanner. Then return to this page to review and
-      manage any Font Awesome versions found.
-    </p>
-    <div>
+      <p>Enable the scanner below and a box will appear in the bottom corner
+      of your window (that only you can see) while it runs for 10 minutes. Then browse
+      your site, especially to the pages having trouble, to
+      catch any Slimers - *ahem* - conflicts in the scanner. Then come back here to review and
```

Comment:
In our call earlier, I noticed that the "ahem" was line-wrapping in the middle of the joke. I think the joke might be more enjoyable if it didn't wrap. So maybe something equivalent to:
`<span style="{{ whiteSpace: "nowrap" }}>Slimers - *ahem* - conflicts</span>`?

Reply:
Yup! 💯